application-passwords.js

```diff
@@ -33,15 +33,23 @@
 				name : name
 			}
 		} ).done( function ( response ) {
+
 			$newAppPassField.prop( 'disabled', false ).val('');
 			$newAppPassButton.prop( 'disabled', false );
 
+			// Check if password was added
+
+			if ( typeof response.error !== 'undefined' ) {
+				alert(response.error);
+				return;
+			}
+
 			$newAppPassForm.after( tmplNewAppPass( {
 				name     : name,
 				password : response.password
 			} ) );
 
-			$appPassTbody.prepend( tmplAppPassRow( response.row ) );
+			$appPassTbody.append( tmplAppPassRow( response.row ) );
```

Comment:
I totally understand but I was very confused from this. When I added a new password it was then prepended to the table.  After I refreshed the page it "moved" from the top of the table to the bottom.  So to keep consistency I changed the `prepend()` to `append()`.  Looking back, and mostly since I am new to this, I realize that this change would be best as its own issue and PR.  Changing random parts of code within a PR is a fast way to developer confusion and lack of consistency so I lesson learned for me :+1: :)